### DISCUSSION TITLE
[Feature Request] Windows 10 start menu pinned tile style transparent (default) and accent color

### DISCUSSION THREAD

**Discussion by XuanChen1234** [valinet/ExplorerPatcher#1673] [Ideas]
We are thinking about Windows 10 start menu pinned tile style transparent (default) and accent color's options,
This is a great restoration project but the Windows 10 start menu background color tiles would be SO much appreciated
If it is not planned yet, I hope this will be a thing in the next update,
Many many thanks, valinet. 
![屏幕截图 2023-07-28 110951](https://github.com/valinet/ExplorerPatcher/assets/140480110/a6778cab-0e24-4674-a80c-55e6be7a2356)

**Comment by vippium** [comment]
not possible as it was not there in initial release of win11..

**Comment by XuanChen1234** [comment]
Windows 10 start menu transparent tile is too ugly, how to change it to background color?

**Reply by XuanChen1234** [reply]
Any more project?

**Comment by XuanChen1234** [comment]
@valinet Any more planned?

If it is not planned yet, I hope this will be a thing in the next update.

**Reply by pyrates999** [reply]
No.  Stop asking.

**Comment by XuanChen1234** [comment]
@valinet Is it possible to backport the Background tiles from Windows 10 20H1 Start menu into ExplorerPatcher?

**Reply by pyrates999** [reply]
Not if windows no longer includes the code that does this.

**Comment by vippium** [comment]
I've already mentioned the reason that why it can't be ported to EP as it was not there in initial Win11 build. And please stop tagging valinet again and again for feature request.

**Comment by pyrates999** [comment]
@XuanChen1234 please close this as it cannot be implemented due to technical limitations.

**Reply by XuanChen1234** [reply]
No! Fuck off!

**Comment by XuanChen1234** [comment]
Any plans for the Windows 10 start menu pinned tile style transparent (default) and background color?

**Comment by XuanChen1234** [comment]
@Jamesstevens-3rd Any plans for the Windows 10 start menu pinned tile style transparent (default) and background color?
